src/raftstore/store/snap.rs

```diff
@@ -557,44 +912,9 @@ mod test {
         // temporary file should not be count in snap size.
         assert_eq!(mgr.rl().get_total_snap_size(), exp_len * 2);
 
-        mgr.rl().get_snap_file(&key2, true).unwrap().delete();
+        mgr.rl().get_snapshot_for_reading(&key2).unwrap().delete();
         assert_eq!(mgr.rl().get_total_snap_size(), exp_len);
-        mgr.rl().get_snap_file(&key2, false).unwrap().delete();
-        assert_eq!(mgr.rl().get_total_snap_size(), 0);
-
-        let mut f4 = mgr.rl().get_snap_file(&key2, true).unwrap();
```

Comment:
In the test case of v1::test, a `size_track` is used to check that the size would be changed for the unsaved Snap. It's not literally the same since the tests are separated.